src/modules/tests/application-reference/application-reference.actions.ts

```diff
@@ -0,0 +1,11 @@
+import { Action } from '@ngrx/store';
+
+export const POPULATE_APPLICATION_REFERENCE = '[JournalEffects] Populate Application Reference';
+
+export class PopulateApplicationReference implements Action {
+  readonly type = POPULATE_APPLICATION_REFERENCE;
+  constructor(public payload: any) {}
```

Comment:
Could you use the `Application` type here?

Reply:
just pushed the update